tools/archetype/binding.test/src/main/resources/archetype-resources/META-INF/MANIFEST.MF

```diff
@@ -7,22 +7,18 @@ Bundle-Version: ${version.replaceAll("-SNAPSHOT", ".qualifier")}
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Fragment-Host: ${package}
 Import-Package: ${package},
- ${package}.internal,
+ ${package}.handler,
  org.slf4j,
- groovy.json,
- groovy.lang,
- org.codehaus.groovy.reflection,
- org.codehaus.groovy.runtime,
- org.codehaus.groovy.runtime.callsite,
- org.codehaus.groovy.runtime.typehandling,
  org.eclipse.smarthome.core.events,
  org.eclipse.smarthome.core.thing.util,
- org.eclipse.smarthome.test,
  org.eclipse.smarthome.test.storage,
+ org.eclipse.smarthome.test.java,
  org.hamcrest;core=split,
+ org.mockito,
  org.junit;version="4.0.0",
  org.osgi.service.device,
  org.osgi.framework
+Require-Bundle: org.junit,org.mockito,org.hamcrest
```

Comment:
@htreu @sjka @kaikreuzer is it expected that thuis creaped in?

The manifest must not contain any `Require-Bundle` entries. Instead, `Import-Package` must be used. [[B.4](http://docs.openhab.org/developers/development/guidelines.html#b-osgi-bundles)]

This means you also have to reference specific packages in the `Import-Package` instead of the top-level packages needed for `Require-Bundle`